code/modules/mob/living/simple_animal/slime/life.dm

```diff
@@ -526,7 +526,7 @@
 		if(prob(2) && prob(t))
 			var/phrases = list()
 			if(Target)
-				phrases += "[Target]... look yummy..."
+				phrases += "[Target]... looks yummy..."
```

Comment:
This one seems somewhat deliberate, I would expect an error like this from a somewhat-sapient slime.

Reply:
I thought so too, but compared to the other language quirks used by the simple slimes, when I see this one in action it keeps striking me as jarring, as if it were an unintentional thing. If that's just me, though, I'm fine with changing it back!

Reply:
I think the slimes should be using more simplistic language like this, although as you said there are probably some inconsistencies

Reply:
Fixed!